src/bors/mod.rs

```diff
@@ -70,6 +71,16 @@ pub trait RepositoryClient: Send + Sync {
     fn get_workflow_url(&self, run_id: RunId) -> String;
 }
 
+/// Temporary trait to sastify the test mocking.
+/// TODO: Remove this trait once we move to mock REST API call.
+#[async_trait]
+pub trait GlobalClient<Client: RepositoryClient>: Send + Sync {
```

Comment:
This is a bit ugly, as I'm sure you also see :D Apart from getting rid of mocking, what about removing the global client from `BorsContext` and only storing it in `consume_global_events` (or someone who calls this function)?

But we don't need to do that, it would require some additional mocking anyway. I would maybe just rename this to `RepositoryLoader`.

Reply:
Yeah we need to mock this to test the installation change event.